docs/changelog-fragments/26.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added sftp get functionality to fetch file
```

Comment:
This is the only place with a sentence in the past tense. All of the others should be changed to be like this.

Reply:
That is because of the PR title.

Reply:
I don't think other places it requires to be changed and it is fine as it is.

Reply:
Fixed, I don't have a strong feeling about this but since you think it is the right away to add changelog I am fine with it 👍

Reply:
Well, then the changelog will be horribly inconsistent.

Reply:
Oh, I see you fixed the other places.

Reply:
P.S. I wasn't talking about changing the PR titles, this would be too much unnecessary work. Besides, PR titles should follow Git commits standard anyway.